utils/data.py

```
@@ -0,0 +1,108 @@
+DATA_DIR = Path(__file__).parent.parent / "data"
+
+
+def load_json(filename):
+    """JSON 파일 읽기"""
+    file_path = DATA_DIR / filename
+
+    if not file_path.exists():
+        return []
+
+    try:
+        with open(file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)